python/lsst/meas/algorithms/objectSizeStarSelector.py

```diff
@@ -430,21 +428,35 @@ def selectStars(self, exposure, sourceCat, matches=None):
         centers, clusterId = _kcenters(width, nCluster=4, useMedian=True,
                                        widthStdAllowed=self.config.widthStdAllowed)
 
-        if display and plotMagSize and pyplot:
-            fig = plot(mag, width, centers, clusterId,
-                       magType=self.config.sourceFluxField.split(".")[-1].title(),
-                       marker="+", markersize=3, markeredgewidth=None, ltype=':', clear=True)
-        else:
-            fig = None
-
```

Comment:
That didn't get removed, just moved to `self._plot()`. The silly `fig=None` was removed, though.